status.go

```diff
@@ -67,6 +67,59 @@ const (
 	versionError                  = "ERROR"
 )
 
+func (s *sequins) serveHealth(w http.ResponseWriter, r *http.Request) {
+	s.dbsLock.RLock()
+
+	status := status{DBs: make(map[string]dbStatus)}
+	for name, db := range s.dbs {
+		status.DBs[name] = copyDBStatus(db.status())
+	}
+
+	s.dbsLock.RUnlock()
+
+	ok := true
+	statuses := make(map[string]map[string]map[string]versionState)
+
+	// Iterate through all of the nodes of the local databases and ensure
+	// that the versions they own are all marked as available. A single
+	// version being in an unvailable state results in an error response.
+	for dbKey, dbValue := range status.DBs {
+		for versionKey, versionValue := range dbValue.Versions {
+			for nodeKey, nodeValue := range versionValue.Nodes {
+				_, exists := statuses[dbKey]
```

Comment:
small nit s/exists/ok  
It is go convention to call the second return from a map "ok"